requirements.txt

```diff
@@ -25,3 +25,6 @@ pyyaml>=5.3,<5.4  # http://pyyaml.org/wiki/PyYAML
 # Only for printing:
 pycups>=1.9,<1.10  # https://pypi.python.org/pypi/pycups
 PyPDF2>=1.26,<1.27  # https://github.com/mstamy2/PyPDF2/blob/master/CHANGELOG
+
+# Only for cmsPrometheusExporter
+prometheus-client>=0.7,<0.8  # https://pypi.org/project/prometheus-client
```

Comment:
Documentation should also be updated at "Installing CMS and its Python dependencies". We need `python3-prometheus-client` for Ubuntu and `python-prometheus_client` for Arch Linux as optonal deps.